gui/default/syncthing/core/syncthingController.js

```diff
@@ -133,7 +135,11 @@ angular.module('syncthing.core')
             }).error($scope.emitHTTPError);
 
             $http.get(urlbase + '/svc/report').success(function (data) {
-                $scope.reportData = data;
+                $scope.reportDataPreview = $scope.reportData = data;
+                if ($scope.system && $scope.config.options.urSeen < $scope.system.urVersionMax) {
```

Comment:
This might not work for candidate releases. For those we set `urAccepted = urVersionMax` (or the equivalent what the variables are a called) on startup, but we don't tweak `urSeen` so this dialog would be triggered unnecessarily.